commands/applycmd.go

```diff
@@ -8,23 +8,30 @@ import (
 
 	"github.com/GoogleContainerTools/kpt/internal/util/setters"
 	"github.com/GoogleContainerTools/kpt/pkg/live"
+	"github.com/GoogleContainerTools/kpt/pkg/live/preprocess"
 	"github.com/spf13/cobra"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/klog"
-	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"sigs.k8s.io/cli-utils/cmd/apply"
+	"sigs.k8s.io/cli-utils/pkg/common"
+	"sigs.k8s.io/cli-utils/pkg/inventory"
 	"sigs.k8s.io/cli-utils/pkg/manifestreader"
 	"sigs.k8s.io/cli-utils/pkg/provider"
 )
 
+const (
+	inventoryPolicyFlag = "inventory-policy"
+	strictPolicy        = "strict"
```

Comment:
Does these consts belong here? Or in a more central location (e.g. `pkg/inventory` or `pkg/common`)

Reply:
Agreed that they should be from some common locations. Moved them as constants in cli-utils.